utils/db.py

```
@@ -64,13 +64,12 @@ def get_hitter_info_by_id(player_id: int):
         return None
         
     try:
-        with connection:
-            with connection.cursor() as cursor:
+        with connection, connection.cursor() as cursor:
                 sql = "SELECT * FROM hitter_info WHERE id = %s"
```

Pylint (3.3.7)

[warning] Bad indentation. Found 16 spaces, expected 12

(W0311)